pkg/logger/logger.go

```diff
@@ -33,4 +33,5 @@ type Logger interface {
 	Successf(format string, a ...interface{})
 	Warningf(format string, a ...interface{})
 	Failuref(format string, a ...interface{})
+	Write(p []byte) (n int, err error)
```

Comment:
We probably need a comment here on why this was added. It can be as simple as something like "added to satisfy `io.Writer` interface.

To add more context: this `interface` probably needs to be re-worked. Forcing the user of this interface to make a decision about which method to use just to get a specific ascii symbol at the front of the log seems like overkill.